net/mwan3/files/usr/sbin/mwan3rtmon

```diff
@@ -1,38 +1,28 @@
 #!/bin/sh
-
 . /lib/functions.sh
+. /lib/functions/network.sh
 . /lib/mwan3/mwan3.sh
 
-LOG="logger -t $(basename "$0")[$$] -p"
-
-clean_up() {
-	$LOG notice "Stopping mwan3rtmon..."
-	exit 0
-}
-
-rtchange() {
-	$LOG info "Detect rtchange event."
-}
-
-main() {
-	local rtmon_interval
-	trap clean_up TERM
-	trap rtchange USR1
+main()
+{
+	local IP family
 
 	config_load mwan3
-	config_get rtmon_interval globals rtmon_interval '5'
-
-	sleep 3
-	while true; do
+	family=$1
+	[ -z $family ] && family=ipv4
+	if [ "$family" == ipv6 ]; then
```

Comment:
Thats a string check? so `=` and `"ipv6"`

Reply:
yes, thank you :)